sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/MeterProviderFactoryTest.java

```diff
@@ -34,65 +46,63 @@ class MeterProviderFactoryTest {
   private final SpiHelper spiHelper =
       SpiHelper.create(MeterProviderFactoryTest.class.getClassLoader());
 
-  @Test
-  void create_Defaults() {
+  @ParameterizedTest
+  @MethodSource("createArguments")
+  void create(MeterProviderModel model, SdkMeterProvider expectedProvider) {
```

Comment:
using the `toString()` for testing seems error prone and fragile, but it's what we have right now....and anything else would understandably be quite difficult.